### PR TITLE
docs: add terms of agreement

### PR DESCRIPTION
**Pull request by H4NLee**

### CODE REVIEW

**Review by youngjoon-lee — APPROVED** [review]
gltm

**Review by gyuguen — APPROVED** [review]
LGTM
We're mixing `${name}`, `<name>` for variables. Let's unite later

**Comment by H4NLee** [comment]
> We're mixing ${name}, <name> for variables. Let's unite later

I intentionally change to <deal_id>, because its type is `uint`. Regardless of the type, is it better to use like "{deal_id}"?

File: .gitbook/1-users/3-data-exchange/2-provide-data.md [hunk 1]
```diff
@@ -76,9 +76,27 @@ Now you can use this certificate in the next step.
...
+  ]
+}
+```
+
 ```bash
-panacead submit-consent ${certificate-file-path} \
+panacead submit-consent ${consent-file-path} \
   --from ${data-provider-addr} \
   --chain-id ${chain-id} \
   --gas auto --gas-adjustment 1.30 --gas-prices 5umed \
```
Comment (gyuguen) [inline .gitbook/1-users/3-data-exchange/2-provide-data.md:102]: This is what I was talking about. :)

**Review by inchori — APPROVED** [review]
lgtm

**Pull request merged**